docs/reference-guides/data/data-core-edit-site.md

```diff
@@ -366,6 +366,18 @@ _Returns_
 
 -   `number`: The resolved template ID for the page route.
 
+### setReusableBlock
```

Comment:
> In the long run it may also be more scalable to have an API more like setEditedEntity rather than adding an action for each entity, so perhaps that should be done for 6.4, or as a follow-up to this PR.

I think I like this idea best. I don't think there'd be any more concern adding `setEditedEntity` rather than `setReusableBlock`. We can leave updating the other actions for a follow-up though.